src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
```

Comment:
According to the task requirements, there should be a separate SCSS file for variables. Consider creating a `_variables.scss` file to store common variables like colors, font sizes, etc., and import it into this file.